apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.html.ts

```diff
@@ -135,492 +134,473 @@ export function StudentAssessmentInstance({
-                                        <td class="text-center">
-                                          ${InstanceQuestionPoints({
-                                            instance_question: instance_question_row,
-                                            assessment_question: instance_question_row, // Required fields are present in instance_question
-                                            component: 'manual',
-                                          })}
-                                        </td>
-                                      `
-                                    : ''}
-                                  <td class="text-center">
-                                    ${InstanceQuestionPoints({
-                                      instance_question: instance_question_row,
-                                      assessment_question: instance_question_row, // Required fields are present in instance_question
-                                      component: 'total',
-                                    })}
-                                  </td>
-                                `
-                              : html`
-                                  ${resLocals.has_auto_grading_question &&
-                                  resLocals.has_manual_grading_question
-                                    ? html`
-                                        <td class="text-center">
-                                          ${formatPoints(instance_question_row.max_auto_points)}
-                                        </td>
-                                        <td class="text-center">
-                                          ${formatPoints(instance_question_row.max_manual_points)}
-                                        </td>
-                                      `
-                                    : ''}
-                                  <td class="text-center">
-                                    ${formatPoints(instance_question_row.max_points)}
-                                  </td>
-                                `}
-                          `
-                        : html`
-                            ${resLocals.has_auto_grading_question
-                              ? html`
-                                  <td class="text-center">
-                                    ${run(() => {
-                                      if (!instance_question_row.max_auto_points) {
-                                        return html`&mdash;`;
-                                      }
-
-                                      // Compute the current "auto" value by subtracting the manual points.
-                                      // We use this because `current_value` doesn't account for manual points.
-                                      // We don't want to mislead the student into thinking that they can earn
-                                      // more points than they actually can.
```

Comment:
You removed this and quite a few other comments in this file. Why?

Reply:
... because I tab completed the migration in that file, oops. In [e1e23d4](https://github.com/PrairieLearn/PrairieLearn/pull/12376/commits/e1e23d46295272acd8e003c590636b6bd659def2) I directly insert the content, no lossy AI in the middle!